chain-impl-mockchain/src/ledger/ledger.rs

```diff
@@ -1023,6 +1037,22 @@ impl Ledger {
 
                 new_ledger = new_ledger_.mint_token(tx.payload().into_payload())?;
             }
+            Fragment::Evm(tx) => {
+                #[cfg(feature = "evm")]
+                {
+                    // WIP: deploying contract
+                    let contract = tx.as_slice().payload().into_payload();
+                    let config = &new_ledger.settings.evm_params.config.into();
+                    let environment = &new_ledger.settings.evm_params.environment;
+                    new_ledger
+                        .evm
+                        .deploy_contract(contract, config, environment)?;
+                }
+                #[cfg(not(feature = "evm"))]
+                {
+                    let _ = tx;
```

Comment:
I think we should not allow any EVM transaction in this case or include some fees anyway otherwise it would be possible to bloat up the storage without having to pay anything